payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -1014,3 +1014,36 @@ constructor(
         }
     }
 }
+
+internal val PaymentMethod.Type.requiresMandateForSetup: Boolean
+    get() = when (this) {
+        PaymentMethod.Type.AuBecsDebit,
```

Comment:
Are we sure we want to automatically add mandates for all of these? On iOS we have a much shorter list. https://github.com/stripe/stripe-ios/blob/5d47acb7ffdb8dd58c871a7ff26e022f2908fff2/StripePayments/StripePayments/Source/API%20Bindings/Models/SetupIntents/STPSetupIntentConfirmParams.swift#L71

Reply:
We return `true` for the same 11 payment method types as iOS.

It might look different because I’m doing extensive matching here for every existing payment method type instead of using an `else` case. Doing this so that we don’t miss anything if we ever add a new payment method type.